drivers/display/ssd1306.c

```diff
@@ -48,6 +48,8 @@ LOG_MODULE_REGISTER(ssd1306, CONFIG_DISPLAY_LOG_LEVEL);
 #define SSD1306_ADDRESSING_MODE		(SSD1306_SET_MEM_ADDRESSING_HORIZONTAL)
 #endif
 
+#define POWER_UP_DELAY_MS 10
```

Comment:
Adding the comment from the PR around how this value was chosen and why the `init` function is blocking would be useful for the future.

Reply:
Yeah fair enough, added a couple lines, happy to change them if it turns out I missed it from the device datasheet.